### PR TITLE
fix

### PR DESCRIPTION
**Pull request by Noanan**

### CODE REVIEW

File: pkg/infra/teams/team3/agent3.go
Comment (MattSScott) [inline pkg/infra/teams/team3/agent3.go:76]: Why aren't the return values used?

**Review by MattSScott — COMMENTED** [review]
Minor thing with return vals

File: pkg/infra/teams/team3/agent3.go
Comment (Noanan) [reply pkg/infra/teams/team3/agent3.go:76]: left them up to @Massoud26's implementation, this PR is just fixing bugs regarding the maps that weren't working

Comment (MattSScott) [reply pkg/infra/teams/team3/agent3.go:76]: sounds fair enough :)

**Review by MattSScott — APPROVED** [review]

File: 13-52-gameLog.csv
Comment (antonypap) [inline 13-52-gameLog.csv]: maybe rm that file before merging it?

**Comment by antonypap** [comment]
Apart from my coment, LGTM

**Pull request merged**
